public/js/components/Charts/Charts.js

```diff
@@ -56,8 +66,25 @@ const Charts = ({ charts, isUpdating, toggleStackChart }) => {
                     />
                 </Col>
             </Row>
+            <Row around='xs'>
+                <Col xs={12} md={8}>
+                    <ScatterChartWrap
+                        hasToggle={false}
+                        error={charts.forkTime.error}
+                        chartType={'ForkTime'}
+                        getClassName={getClassName}
+                        filterVals={filterVals}
+                        titleHeader={renderTitle('forkTime')}
+                        hasCsvButton={true}
+                        scale='time'
+                        data={charts.forkTime.data}
+                        height={250}
+                        yLabel='Time since forkage'
```

Comment:
This should probably be `Time since fork` I don't think editorial will be familiar with `forkage`

Reply:
haha this is just my bad english